app/black_spots/views.py

```diff
@@ -78,14 +81,12 @@ def get_queryset(self):
 
 
 class EnforcerAssignmentViewSet(drf_mixins.ListModelMixin, viewsets.GenericViewSet):
-    """
-    ViewSet for enforcer assignments
+    """ViewSet for enforcer assignments"""
+    # Enforcer assignments do not currently have an associated model in the database, and are
```

Comment:
Good question, glad you asked. DRF adds any ViewSet docstrings to the endpoint description in the browseable API. Since this comment is specific to the code implementation, it doesn't really belong in API docs.